### PR TITLE
Added Kuaikanmanhua support

### PR DESCRIPTION
**Pull request by hypoflex**
✓	 constructor
✗	_getMangaList => missing async support
✓	_getMangaListFromPages
✓	_getMangaFromURI
✓	_getChapterList
✗	_getPageList
✓	favicon => missing in commit, added for next commit
✓	Lint
✓	Test

was unable to grab the image (for now)
rest can be tested. images gathering will continue later.

first sleep.

### CODE REVIEW

File: src/web/mjs/connectors/kuaikanmanhua.mjs [hunk 1]
```diff
@@ -81,15 +77,9 @@ export default class Kauikanmanhua extends Connector {
...
-            let data = await this.fetchDOM(request, 'div.entry-content noscript');
-            let copy = [];
-            data.forEach(function (item) {
-                //create new variable to work with
-                let str = item.innerText;
-                //push only the url of "src" into the copy array
-                copy.push(str.substring(str.lastIndexOf("src='") + 5, str.lastIndexOf("g'") + 1));
-            });
-            let pageList = copy.map(element => this.getAbsolutePath(element, request.url));
+            let data = await this.fetchDOM(request, 'div.imgList source');
```
Comment (27136773) [inline src/web/mjs/connectors/kuaikanmanhua.mjs]: Try fetchUI instead to get it from a headless browser window

```javascript
let script = `
new Promise(resolve => {
    let pages = __NUXT__.data[0].comicInfo.comicImages.map(img => img.url);
    resolve(pages);
});
`;
let request = new Request(this.url + chapter.id, this.requestOptions);
let pageList = await Engine.Request.fetchUI(request, script);
callback(null, pageList);
```

Comment (hypoflex) [reply src/web/mjs/connectors/kuaikanmanhua.mjs]: I'll look into it, thanks for the tip!

**Comment by hypoflex** [comment]
✓	 constructor
✗	_getMangaList => missing async support
✓	_getMangaListFromPages
✓	_getMangaFromURI
✓	_getChapterList
✓	_getPageList
✓	favicon => missing in commit, added for next commit
✓	Lint
✓	Test

_getMangaList is missing async support. something i can look into but the not high priority

ready for review :)

**Review by 27136773 — APPROVED** [review]
Added a small fix to hide locked chapters

**Pull request merged**